pkg/sentry/syscalls/linux/sys_time.go

```diff
@@ -121,8 +121,10 @@ func getClock(t *kernel.Task, clockID int32) (ktime.Clock, error) {
 	switch clockID {
 	case linux.CLOCK_REALTIME, linux.CLOCK_REALTIME_COARSE:
 		return t.Kernel().RealtimeClock(), nil
-	case linux.CLOCK_MONOTONIC, linux.CLOCK_MONOTONIC_COARSE, linux.CLOCK_MONOTONIC_RAW:
+	case linux.CLOCK_MONOTONIC, linux.CLOCK_MONOTONIC_COARSE,
+		linux.CLOCK_MONOTONIC_RAW, linux.CLOCK_BOOTTIME:
 		// CLOCK_MONOTONIC approximates CLOCK_MONOTONIC_RAW.
+		// CLOCK_BOOTTIME is equivalent to CLOCK_MONOTONIC, which already include save/restore time
```

Comment:
Comments in Go should end in a period.

Could you expand this comment to provide a more a complete an explanation? That CLOCK_BOOTIME is CLOCK_MONOTONIC plus time in suspend. gVisor has no suspend, and time in the closest equivalent (S/R) is already included in the monotonic clock.

Reply:
Done, this should provide a better explanation.